internal/datastore/crdb/migrations/zz_migration.0007_add_relation_index_storing_caveat.go

```diff
@@ -0,0 +1,27 @@
+package migrations
+
+import (
+	"context"
+
+	"github.com/jackc/pgx/v5"
+)
+
+const (
+	addRelationshipIndexStoringCaveatQuery = `
+		CREATE INDEX idx_relationship_caveat_storing ON relation_tuple (namespace, relation, object_id, userset_relation) STORING (caveat_name, caveat_context);
```

Comment:
Is there a reason you've omitted `userset_namespace` and `userset_object_id` from the index? 

This may also be better as an `ALTER` to add the caveat storage to an existing index.

Reply:
Ok, thanks